galaxy_ng/app/migrations/0053_create_system_auditor_role.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.13 on 2024-06-06 19:13
+
+from django.db import migrations
+
+
+def create_system_auditor_role(apps, schema_editor):
+    Role = apps.get_model("core", "Role")
+    Permission = apps.get_model("auth", "Permission")
+
+    # Create the role
+    role, created = Role.objects.get_or_create(
+        name='galaxy.system_auditor',
```

Comment:
Philosophically, reading into the intent behind the naming here, `galaxy.*` suggests to me this is _not_ intended to be a shared role. Practically, we need an auditor role to be a shared role. For models and permissions, we name things with `shared.*`. So maybe you could name it `shared.system_auditor` or `shared.cluster_auditor` or something like that. That would suggest that naming of other roles would be like `shared.organization_admin`, which I have not looked into yet.

If you're introducing a auditor role that is _not_ shared, then I don't have anything to add, other than that's not a requirement coming from my side.

Reply:
@AlanCoding this is a pulp thing. We have to prefix our roles with `galaxy.`.